internal/cmd/genop/genop.go

```diff
@@ -196,6 +196,16 @@ func {{$name}}Const(n *node) {
 		{{- if $op.Shift}}
 		v := constant.Shift(vConstantValue(v0), token.{{tokenFromName $name}}, uint(vUint(v1)))
 		n.rval.Set(reflect.ValueOf(v))
+		{{- else if (eq $op.Name "/")}}
+		// TODO(mpl): exclude uints?
+		var operator token.Token
+		if n.typ.untyped && isInt(n.typ.rtype) {
+			operator = token.QUO_ASSIGN
```

Comment:
Can you add a comment explaining that `token.QUO_ASSIGN` forces a integer division, as a special case in `constant` package, as it is not obvious when reading the code ?

Reply:
done